examples/ecm/fastapi_app.py

```
@@ -0,0 +1,39 @@
+from open_fdd.ecm_engineering import ECMJob
+
+app = FastAPI(title="Open-FDD ECM Workbook Service")
+
+class ECMRequest(BaseModel):
+    name: str = "Open-FDD ECM Job"
+    globals: dict[str, Any] = Field(default_factory=dict)
+    ecms: list[dict[str, Any]] = Field(default_factory=list)
+
+@app.post("/ecm-workbook")
+def create_ecm_workbook(request: ECMRequest):
+    output_dir = Path("/tmp/open_fdd_ecm")
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/open_fdd_ecm"

(S108)